Squeezer/src/main/res/values/styles.xml

```diff
@@ -84,12 +80,15 @@
     <style name="SqueezerTextAppearance.Large" parent="@android:style/TextAppearance.Large"/>
     <style name="SqueezerTextAppearance.Medium" parent="@android:style/TextAppearance.Medium"/>
     <style name="SqueezerTextAppearance.Small" parent="@android:style/TextAppearance.Small"/>
+    <style name="SqueezerTextAppearance.ContextMenu" parent="@style/TextAppearance.AppCompat.Menu"/>
 
     <!-- List headers -->
     <style name="SqueezerTextAppearance.ItemList.SubHeader" parent="SqueezerTextAppearance.Small"/>
 
     <!-- Text in list items -->
-    <style name="SqueezerTextAppearance.ListItem.Primary" parent="SqueezerTextAppearance.Medium"/>
+    <style name="SqueezerTextAppearance.ListItem.Primary" parent="SqueezerTextAppearance.Medium">
+        <item name="android:textSize">16sp</item>
```

Comment:
I would prefer to use standard sizes. E.g., the only difference between `@android:style/TextAppearance.Medium` and `@android:style/TextAppearance` is 18sp -> 16sp.
Also, please share your thoughts about the selected font sizes, e.g., why use 16sp in some lists and 18sp in others, and why 12sp in now playing mini?

Reply:
I've adjusted now playing mini to 14/14, just with different colors. That adjustment solves the idea of looking less crammed even better.